docs/src/development/file-transfer.md

```diff
@@ -4,34 +4,36 @@ weight: 7
 sidebarTitle: File Transfer
 ---
 
-After your app is built, its file system is read-only. 
-This means that the only way you can edit your app's code is through Git.
+After your app is built, its file system is read-only.
 
-However, you can transfer files to and from your app through mounts.
-To do so, follow these steps.
+The only way you can transfer files to your built app without using Git is through [mounts](../create-apps/app-reference.md#mounts).
+Mounts let you set up directories that remain writable after the build is complete.
+You can then upload files directly to mounts inside your app.
+
+To download files from your built app, 
+you can also use mounts or [an alternative data export method](../tutorials/exporting.md).
```

Comment:
Actually, there isn't much information on alternative methods to download files from a built app in the Exporting data tutorial (apart from the `project:get` command that downloads the whole project). Are there any other specific ways you were thinking of and that I should explicitly mention or link to?

Reply:
Introduction reworked